sample-android/src/main/java/com/santarest/sample/MainActivity.java

```diff
@@ -57,44 +58,52 @@ public void onResponseReceived(Object action, Request request, Response response
                 })
                 .build();
         uploadFileServer.sendAction(new UploadFileAction());
+        githubRest.observeActions()
+                .ofType(ExampleAction.class)
+                .subscribe(new Action1<ExampleAction>() {
+                    @Override
+                    public void call(ExampleAction exampleAction) {
+                        System.out.println("exampleAction = [" + exampleAction + "]");
+                    }
+                });
         githubRest.sendAction(new ExampleAction("square", "otto"));
```

Comment:
I think it will need in a branch to improve sample :) 
Please, tell me comments for changes on only this branch